### ISSUE TITLE
What do your table manners say about you?

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#864]
https://www.bbc.co.uk/ideas/videos/what-do-your-table-manners-say-about-you/p07gbkp3

**Comment by nelsonic** [comment]
Need to buy a book on this so that I can read it and correct anything I'm doing "wrong" so that I don't pass on any bad habits to the next generation. Manners matter more than we realise. 💭

**Comment by iteles** [comment]
We own Sandy Toksvig's Peas and Queues btw: https://books.google.pt/books?id=UYT4AAAAQBAJ&pg=PP7&hl=pt-PT&source=gbs_selected_pages&cad=3#v=onepage&q&f=false

I think the issue for me isn't the theory, it's the practice 😬🙃

**Comment by nelsonic** [comment]
Yeah, Peas and Queues is more of a humorous history lesson rather than a concise field manual. 
Totally worth reading but at [272 pages](https://www.amazon.co.uk/Peas-Queues-Minefield-Modern-Manners/dp/1781250324) 📚 I don't know how many people will pick it up ...

I was thinking something a bit quicker to read:

A Kids' Guide to Manners: 50 Fun Etiquette Lessons for Kids (and Their Families):
https://www.amazon.co.uk/Kids-Guide-Manners-Etiquette-Families/dp/1641520957
![image](https://user-images.githubusercontent.com/194400/174567271-258f6ce2-e1d1-42c1-a60a-31c561e40e20.png)

Modern Manners: Tools to Take You to the Top
https://www.amazon.co.uk/Modern-Manners-Dorothea-Johnson/dp/0770434088
![image](https://user-images.githubusercontent.com/194400/174567438-296a7adc-cb1b-4dc0-b913-5d7e2042d1a3.png)

As for _theory_ vs. _practice_ ... re-reading the "rules" might be helpful. 📝
